dev-tools/packaging/packages.yml

```diff
@@ -433,6 +433,7 @@ shared:
       docker_entrypoint: 'docker-entrypoint.elastic-agent.tmpl'
       user: '{{ .BeatName }}'
       linux_capabilities: ''
+      image_name: ''
```

Comment:
Forced to redefine this particular arg, otherwise:

```
 => => naming to docker.elastic.co/beats/elastic-agent-cloud:8.0.0                                                                                                                                             0.0s

Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
package ran for 2m12.830616488s
Error: failed building elastic-agent type=docker for platform=linux/amd64: failed to prepare build: expanding template '/Users/vmartinez/work/src/github.com/elastic/beats/dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl' to 'build/package/elastic-agent/elastic-agent-linux-amd64.docker/docker-build/Dockerfile': failed to expand template: template: /Users/vmartinez/work/src/github.com/elastic/beats/dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl:26:16: executing "/Users/vmartinez/work/src/github.com/elastic/beats/dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl" at <.image_name>: map has no entry for key "image_name"
```